py/kubeflow/kubeflow/cd/access_management.py

```diff
@@ -0,0 +1,44 @@
+""""Argo Workflow for building Access Management's OCI image using Kaniko"""
```

Comment:
I feel like you can abstract this Builder python file given you have multiple almost same file here, add it in e.g, `py/kubeflow/kubeflow/cd/base_builder.py`

And for each Argo Workflow, just reference it should be fine